packages/instant/webpack.config.js

```diff
@@ -135,6 +135,9 @@ const generateConfig = (dischargeTarget, heapConfigOptions, rollbarConfigOptions
         resolve: {
             extensions: ['.js', '.json', '.ts', '.tsx'],
         },
+        node: {
+            fs: 'empty',
+        },
```

Comment:
@fragosti @BMillman19 see any issues with doing this?

Reply:
chatted on Slack, seems like it will be OK. 

for context,  this tells webpack to stub `fs` with an empty module.